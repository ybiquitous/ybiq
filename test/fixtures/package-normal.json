{
  "scripts": {
    "test": "abc",
    "lint:js": "eslint ."
  },
  "lint-staged": {
    "*.css": "xyz"
  },
  "commitlint": {
    "extends": []
  }
}
