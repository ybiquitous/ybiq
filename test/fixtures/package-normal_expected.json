{
  "scripts": {
    "test": "abc",
    "lint:js": "eslint --ignore-path .gitignore --ext .js,.jsx,.mjs,.ts,.tsx .",
    "test:watch": "abc --watch",
    "test:coverage": "echo \"unsupported.\" && exit 1",
    "lint:js:fix": "npm run lint:js -- --fix",
    "lint:md": "remark . --frail",
    "lint:md:fix": "remark . --output",
    "lint": "npm-run-all --print-label --parallel lint:*",
    "prettier": "prettier --ignore-path .gitignore .",
    "prettier:check": "npm run prettier -- --check",
    "prettier:write": "npm run prettier -- --write",
    "format": "npm-run-all --print-label --parallel lint:*:fix prettier:write",
    "prerelease": "git checkout master && git pull origin master && npm ci && npm test",
    "release": "standard-version",
    "release:dry-run": "standard-version --dry-run"
  },
  "husky": {
    "hooks": {
      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "*.css": "xyz",
    "*.{js,jsx,mjs,ts,tsx}": "eslint --fix --no-ignore",
    "*": "prettier --write",
    "!(CHANGELOG).md": "remark --frail"
  },
  "standard-version": {
    "sign": true,
    "scripts": {
      "postchangelog": "prettier --write CHANGELOG.md"
    }
  },
  "remarkConfig": {
    "plugins": [
      "preset-lint-recommended",
      "lint-no-heading-punctuation",
      ["lint-list-item-bullet-indent", false],
      ["lint-list-item-indent", false],
      "validate-links"
    ]
  },
  "commitlint": {
    "extends": ["@commitlint/config-conventional"]
  },
  "eslintConfig": {
    "extends": ["ybiquitous"]
  }
}
